test/performance/performance-tests.sh

```diff
@@ -53,7 +53,7 @@ function update_benchmark() {
   kubectl patch configmap config-mako -n "${TEST_NAMESPACE}" -p '{"data":{"environment":"prod"}}' || abort "failed to patch config-mako configmap"
 
   echo ">> Updating benchmark $1"
-  ko delete -f "${benchmark_path}"/${TEST_CONFIG_VARIANT} --ignore-not-found=true
+  ko delete -f "${benchmark_path}"/${TEST_CONFIG_VARIANT} --ignore-not-found=true --wait=false
```

Comment:
can there be a problem if you are applying below while also deleting? This makes it the call to ko non-blocking?

Reply:
Hmm, I'm actually not sure. I thought `ko apply` could have eventual consistency but I'm probably wrong.

Reply:
I tried to run with this change for 5 times, and they all succeeded, so this change should be safe.